hushline/settings/__init__.py

```diff
@@ -605,6 +616,38 @@ def update_smtp_settings() -> Response | str:
         flash("👍 SMTP settings updated successfully")
         return redirect(url_for(".index"))
 
+    @bp.route("/update-brand-primary-color", methods=["POST"])
+    @admin_authentication_required
+    @personal_server_required
+    def update_brand_primary_color() -> Response | str:
+        host_org = HostOrganization.fetch_or_default()
+        form = UpdateBrandPrimaryColorForm()
+        if form.validate_on_submit():
+            host_org.brand_primary_hex_color = form.brand_primary_hex_color.data
+            db.session.add(host_org)  # explicitly add because instance might be new
+            db.session.commit()
+            flash("👍 Brand primary color updated successfully.")
+            return redirect(url_for(".index"))
+
+        flash("⛔ Invalid form data. Please try again.")
+        return redirect(url_for(".index"))
+
+    @bp.route("/update-brand-app-name", methods=["POST"])
+    @admin_authentication_required
+    @personal_server_required
+    def update_brand_app_name() -> Response | str:
+        host_org = HostOrganization.fetch_or_default()
```

Comment:
I think this revision may swallow DB inconsistency errors during runtime. Perhaps the original with your added ``fetch`` instead of the previous ``default``?

https://github.com/scidsg/hushline/blob/06042b73b1b8367c9a117296d3addc114357e772/hushline/settings.py#L742-L744

Reply:
I don't think we need to error here because if the row isn't present, then the app would be busted in many other places and the user wouldn't even be able to navigate to this page in the first place. Like we don't `try/except` on the DB query itself for `OperationalError`s either, just assume that's live and let the top level error handler deal with that.